src/functions/Functions.php

```diff
@@ -50,7 +50,6 @@ public function sayHello(): string
     {
         return 'Hello';
     }
-
```

Comment:
only remove extra empty line, there should aways be one empty line between methods